npm/esbuild-linux-loong64/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "esbuild-linux-loong64",
```

Comment:
Note that this package name doesn't work because it's already in use, and already has content published to it: https://www.npmjs.com/package/esbuild-linux-loong64. I will have to come up with another package name instead.

Reply:
Hi,

"esbuild-linux-loong64" sounds like a good name, Whether can contact [npmjs/esbuild-linux-loong64 package](https://www.npmjs.com/package/esbuild-linux-loong64) ,
 ask him to help deal with this package

cheers~

Reply:
> Note that this package name doesn't work because it's already in use, and already has content published to it: https://www.npmjs.com/package/esbuild-linux-loong64. I will have to come up with another package name instead.

I have deleted this package.